src/test/java/core/basesyntax/strategy/impl/FruitTransactionStrategyTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.FruitTransactionHandler;
+import core.basesyntax.strategy.FruitTransactionStrategy;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionStrategyTest {
+    private static FruitTransactionStrategy fruitTransactionStrategy;
+    private static FruitTransaction fruitTransaction;
+    private static FruitDao fruitDao;
+    private FruitTransactionHandler actual;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitTransactionStrategy = new FruitTransactionStrategyImpl();
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Before
+    public void setUp() {
+        FruitStorage.storageFruits.put("banana", 100);
+    }
+
+    @Test
+    public void chooseBalanceHandler_ok() {

```

Comment:
```suggestion
    public void getTransaction_chooseBalanceHandler_ok() {
```
the same for other test methods in this class